qcoptions.py

```diff
@@ -1,5 +1,12 @@
-from gempython.utils.standardopts import parser
+from optparse import OptionParser
```

Comment:
Is everything that uses `qcoptions` happy with this change?

Reply:
The only one who is "not" is `ultraLatency.py` since communication with `amc13` requires knowledge of the shelf parameter.  This has been added as a dedicated argument for `ultraLatency.py`.

So everything is fine.